### ISSUE TITLE
Q. How to create a per ROM controller config 

### ISSUE THREAD

**Issue by sym-ion** [m64p/mupen64plus-GLideN64#74]
Build: 05-02-2018 
OS: Windows 10
Plugin: mupen64plus-input-sdl.dll
Issue: Likely user-error but none of the standard methods of making a particular controller config for a single ROM have worked for me. Didn't find a simple guide upon searching to do so.

**Comment by loganmc10** [comment]
Yes this is something that is currently isn't supported. There is no way currently to set options per-game, besides some settings that are stored in mupen64plus.ini and GLideN64.custom.ini, but there is no way currently to set controller options per game
